docs/diagrams/StorageDiagram.puml

```diff
@@ -9,6 +8,7 @@
 Package " "<<Rectangle>>{
 
     Class Storage STORAGE_COLOR
+    Class Family MODEL_COLOR_T2
     Class PersonList MODEL_COLOR
     Class Person MODEL_COLOR
     Class Money MODEL_COLOR_T3
```

Comment:
Slight logic error here, each `Person` has `1` `IncomeList` and `1` `ExpenditureList`. There is no `Money` class in between